src/app/base/components/node/TestForm/TestForm.test.tsx

```diff
@@ -72,54 +66,64 @@ describe("TestForm", () => {
     });
   });
 
-  it("calls the action to test given machines", () => {
+  it("calls the action to test given machines", async () => {
     const store = mockStore(state);
```

Comment:
```suggestion
    // load only the internet connectivity script
    const script = state.script.items[1];
    state.script.items = [state.script.items[1]];
    const store = mockStore(state);
```

Reply:
Technically we could load both scripts, but a single script is sufficient for the purpose of this test.